python/ql/src/semmle/python/Files.qll

```diff
@@ -341,7 +341,12 @@ abstract class Container extends @container {
     predicate isStdLibRoot(int major, int minor) {
         allowable_version(major, minor) and
         this.isImportRoot() and
-        this.getBaseName().regexpMatch("python" + major + "." + minor)
+        (
+            this.getBaseName().regexpMatch("python" + major + "\\.?" + minor)
```

Comment:
Should this allow `Python` as well, or does the upper-case variant only ever occur inside a `lib` directory?